src/assets/scss/application.scss

```diff
@@ -235,3 +235,7 @@
   margin-bottom: 30px;
   margin-top: 30px;
 }
+
+#govuk-template_mobile {
```

Comment:
Hi @SophiaDWokoma - minor point but I think the convention for BEM modifiers is double underscore (i.e. `govuk-template__mobile`)

Reply:
Updated

Reply:
updated